downstream/modules/aap-on-azure/proc-peer-vwan-hub-to-aap.adoc

```diff
@@ -0,0 +1,33 @@
+[id="proc-peer-vwan-hub-to-aap"]
+
+= Peering a VWAN Hub to the {AAPonAzureNameShort} Network
+
+Before you can peer a VWAN with the {AAPonAzureNameShort} VNET, you must have connected a hub network and at least one spoke network to the hub network of the Azure VWAN that you intend to connect {AAPonAzureNameShort} to.
+
+.Prerequisites
+
+* A pre-configured Azure VWAN that is already tested and working.
+* One or more of the following connections to the VWAN:
+** A DMZ network that contains Azure virtual machines that users will use to access {AAPonAzureNameShort}.
+** A DMZ network that contains an Azure virtual machine that can be used with SSH tunneling to access {AAPonAzureNameShort}.
+** A VPN or Direct Connect service to your local network that routes traffic from machines on your local network to {AAPonAzureNameShort}.
+
+Peering the {AAPonAzureNameShort} VNET to your VWAN follows the same approach as peering any other VNET or network with a VWAN.
+
+. Navigate to the *Virtual Network Connections* page for the VWAN that you want to peer with your {PlatformNameShort} instance.
+. Use the following settings to create a connection between the VWAN hub and your {PlatformNameShort} instance:
+** *Connection Name*: _<Ansible_Automation_Platform_connection_name>_
+** *Hubs*: Select one or more VWAN hub networks that the managed application VNET will peer with.
+** *Subscription*: Select the subscription where {AAPonAzureNameShort} has been deployed.
+** *Resource group*: The managed resource group of the managed application. It will typically be prefixed with “mrg-”.
+** *Virtual network*: The VNET of the managed application. There is only one VNET in the managed resource group.
+** *Propagate to none*: _No_
+** *Associate Route Table*: Select the default route table or the appropriate route table that your organization has configured for VWAN. 
+** *Propagate to Route Tables*: Select one or more default route tables or the appropriate route table that your organization has configured for VWAN. 
+** *Propagate to labels*: Select labels if your organization uses them.
+** *Static routes*: Do not complete this field.
```

Comment:
This is seems like a bit of a mix of reference and procedure. It looks like an enumeration of all the settings coupled with the action that users should take.

How about making each one a substep with a well-defined action?

.. Specify the _<Ansible_Automation_Platform_connection_name>_ with the *Connection Name* field.
.. From the *Hubs* field, select one or more ...
...
.. Do not enter any value in the *Static routes* field.

Reply:
I agree.  
The Azure team specifically didn't want a proper procedure because it describes the Azure UI, which Red Hat don't support.
So all the peering procedures are along the lines of "Navigate to the correct page and enter these values in the following fields in the form".

Reply:
@oraNod I'll make this change in another PR, as I'll have to update the other peering procedures to be consistent with this one. The PM has some updates for various procedures, so I'll run this change by him at the same time.